code/client/munkilib/appleupdates.py

```diff
@@ -640,7 +640,7 @@ def _GetPreferredLocalization(self, list_of_localizations):
         localizations. Returns a single language/localization name.'''
         # pylint: disable=no-self-use
         localization_preferences = (
-            munkicommon.pref('AppleSoftwareUpdateLanguages') or ['English'])
+            munkicommon.getAppleLanguages() or ['English'])
```

Comment:
I understand that. This change won't really help that at all.

Even if it operated as (I think) intended, if Johann is logged in when managedsoftwareupdate is running in the background, Munki will get German dist files. Then Johann logs out and Jean-Louis logs in. The .dist files are still in German, at least until the next background run (and maybe even then).

Much much much more work will need to be done to correctly support the use-case of multiple users with different languages on one Mac. It's not a need I have, so I'm not really willing to go into all the effort.